src/pages/auth/register.tsx

```diff
@@ -140,6 +136,116 @@ const Page = () => {
                 name="password"
                 type="password"
               />
+              <Field
+                as={Select}
+                label="Country"
+                name="country"
+                fullWidth
+                onChange={(_: SelectChangeEvent, element: ReactElement) => {
+                  const country = element.props.value;
+                  setFieldValue("country", element.props.value);
+
+                  if (country == "US") {
+                    setFieldValue("useCase", UseCase.EmbeddedFinance);
+                  } else {
+                    setFieldValue("useCase", UseCase.ExpenseManagement);
+                  }
+                }}
+              >
+                <MenuItem value="AT">Austria</MenuItem>
+                <MenuItem value="BE">Belgium</MenuItem>
+                <MenuItem value="HR">Croatia</MenuItem>
+                <MenuItem value="CY">Cyprus</MenuItem>
+                <MenuItem value="EE">Estonia</MenuItem>
+                <MenuItem value="FI">Finland</MenuItem>
+                <MenuItem value="FR">France</MenuItem>
+                <MenuItem value="DE">Germany</MenuItem>
+                <MenuItem value="GR">Greece</MenuItem>
+                <MenuItem value="IE">Ireland</MenuItem>
+                <MenuItem value="IT">Italy</MenuItem>
+                <MenuItem value="LV">Latvia</MenuItem>
+                <MenuItem value="LT">Lithuania</MenuItem>
+                <MenuItem value="LU">Luxembourg</MenuItem>
+                <MenuItem value="MT">Malta</MenuItem>
+                <MenuItem value="NL">Netherlands</MenuItem>
+                <MenuItem value="PT">Portugal</MenuItem>
+                <MenuItem value="SK">Slovakia</MenuItem>
+                <MenuItem value="SI">Slovenia</MenuItem>
+                <MenuItem value="ES">Spain</MenuItem>
+                <MenuItem value="GB">United Kingdom</MenuItem>
+                <MenuItem value="US">United States</MenuItem>
+              </Field>
+              {/* <Field
+                as={Select}
+                label="Use case"
+                name="useCase"
+                fullWidth
+                error={!!(touched.useCase && errors.useCase)}
+              >
+                <MenuItem value={UseCase.EmbeddedFinance}>
+                  Embedded Finance
+                </MenuItem>
+                <MenuItem value={UseCase.ExpenseManagement}>
+                  Expense Management
+                </MenuItem>
+              </Field> */}
+
+              <Divider />
+              <FormLabel>What kind of app are you building?</FormLabel>
```

Comment:
"app" is a bit reductive, but i can't find a more concise term for "issuing-and-maybe-treasury program"